doc/api/users.md

```diff
@@ -150,6 +150,51 @@ Parameters:
 }
 ```
 
+### Get SSH key with user by ID of an SSH key
```

Comment:
This should probably be moved into a separate api/keys doc since it's no longer about users.